### ISSUE TITLE
Double quote error for file function in interpolation

### ISSUE THREAD

**Issue by killercentury** [VladRassokhin/intellij-hcl#6]
When I use the built-in file function in interpolation syntax as following
```
container_definitions = "${file("ecs-container-definitions.json")}"
```
Terraform commands could accept it without problem. But IntelliJ reports as syntax error. This syntax does look weird, but the official docs also use in this way.

**Comment by VladRassokhin** [comment]
Will investigate next week.
Also, do you use syntax like `size = 10Gb`? Is that correct in terraform files?

**Comment by VladRassokhin** [comment]
Fixed, update to 0.2.3
